content/docs/faq-styling.md

```diff
@@ -28,24 +28,24 @@ render() {
 }
 ```
 
->Tip
->
->If you often find yourself writing code like this, [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs) package can simplify it.
+>小技巧
+>  
```

Comment:
Unnecessary empty

Reply:
I think empty on line 32 was on the original file too.